qiskit/qasm/node/nodeerror.py

```diff
@@ -13,7 +13,7 @@
 """Exception for errors raised while interpreting nodes."""
 
 
-class NodeException(Exception):
+class NodeError(Exception):
```

Comment:
API break - we need to revert this.

Reply:
This was reverted a few days ago (I only uncovered this batch of your comments a couple of hours ago.)